srcdeps-maven-enforcer/src/test/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcerTest.java

```diff
@@ -0,0 +1,233 @@
+/**
+ * Copyright 2015-2017 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.srcdeps.mvn.enforcer;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Properties;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.srcdeps.core.config.MavenAssertions;
+
+public class SrcdepsEnforcerTest {
+
+    @Test
+    public void assertFailWith() {
```

Comment:
I'm not quite sure what this is doing. Would it be worth adding a javadoc, or a comment?